tests/Feature/PetTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Pet;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class PetTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testStorePetCreatesPet(): void
```

Comment:
This naming is a bit unreadable, consider something like `testUserCanCreatePet`.